### ISSUE TITLE
Suggesting a few extra experiments

### ISSUE THREAD

**Issue by 2sang** [HanbumKo/MI333_assignment1#1]
### Suggests a few extra experiments
1. (auroc.py) With the existing pre-trained weights, add classification accuracy only w.r.t the test images that have labels between 0 and 5(in-distribution)
2. (auroc.py) Replace the sigmoid activation function in the last layer to **softmax**.
Reproduce the plots and auroc value, roc curve, accuracy(under the same testing condition as *1.* above) as well
3. Bring the previous experiment which used the sigmoid activation function, compare the set sof consequences with the experiment done from *2.*.

**Comment by HanbumKo** [comment]
I found my mistake. And I found most of logistic regression don't use activation function in last layer. I revised the codes and test 1, 2, 3 which is you mentioned above.
